app/admin/views/items.py

```diff
@@ -0,0 +1,84 @@
+from flask_admin.form import rules
+from flask_admin.model.base import BaseView as FlaskBaseView
+from flask_sqlalchemy import BaseQuery
+from jinja2.runtime import Context
+from sqlalchemy.orm.scoping import scoped_session
+
+from .base import BaseView
+from ...models import Items, Storages, Users
+from ...utils.swagger_models import ItemsModels
+from ...utils.views import QuerySelectField
+
+
+def _get_user_label(user: Users) -> str:
+    label = f'{user.user_id}'
+    for value in [user.name, user.surname]:
+        if value is not None:
+            label += f' {value}'
+    return label
+
+
+def _get_storages_query() -> BaseQuery:
+    return Storages.query.order_by(Storages.storage_id)
+
+
+def _get_storage_label(storage: Storages) -> str:
+    label = f'{storage.storage_id}'
+    return label
+
+
+def _render_owner(view: FlaskBaseView, context: Context, model: Items, name: str) -> str:
+    storage = Storages.query.filter_by(storage_id=model.storage_id).first()
+    return _get_user_label(Users.query.filter_by(user_id=storage.user_id).first())
```

Comment:
It could be better to use only one request to the database from Python. For example, you can use a subquery:
```py
    storage = Storages.query.filter_by(storage_id=model.storage_id).subquery()
    return _get_user_label(Users.query.filter_by(user_id=storage.c.user_id).first())
```